com.provar.plugins.provardx/src/commands/provar/metadatacache.ts

```diff
@@ -31,7 +32,7 @@ export default class metadatacache extends SfdxCommand {
     
   protected static flagsConfig = {
     // flag with a value (-f, --propertyfile=VALUE)
-    metadatalevel: flags.string({char: 'm', description: messages.getMessage('metadataLevelFlagDescription')}),
+    caching: flags.string({char: 'm', description: messages.getMessage('metadataLevelFlagDescription')}),
```

Comment:
name should remain metadatalevel only, instead update schema and property file (replacing caching with metadatalevel)